### ISSUE TITLE
Merge --collapse-imports-closure method does not work as expected

### ISSUE THREAD

**Issue by cmungall** [ontodev/robot#249]
Related to #84 

Looks like I forgot to pass on the options in https://github.com/ontodev/robot/commit/130c11c1abec2fa1fbbfead84cb7fc5b5ff9a09b

**Comment by rctauber** [comment]
This was fixed by #277

**Comment by rctauber** [comment]
This should be resolved, but I need to change this line -
```
Set<OWLOntology> imports = targetOntology.getImports();
```

To -
```
Set<OWLOntology> imports = targetOntology.getDirectImports();
```

If an ontology has indirect imports, they are added as declarations in the output when `--collapse-import-closure false`. Other than that, it seems to be working as expected. Let me know if you see any other issues with this.

**Comment by rctauber** [comment]
This should be resolved by #341 - please re-open if there are any other issues with it.
